### PR TITLE
initial version

### PR DESCRIPTION
**Pull request by erkangz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,60 @@
 # graph-nmap
-Ingest hosts discovered from nmap scans to JupiterOne graph
+
+This is a local/unmanaged integration that scans vulnerability reports from
+[Vuls.io](https://vuls.io) scans and ingests the findings into JupiterOne (J1).
```
Comment (aiwilliams) [inline README.md:4]: This is referring to vuls.io.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,60 @@
...
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your
+local machine for development and testing purposes. See deployment for notes on
+how to deploy the project on a live system.
```
Comment (aiwilliams) [inline README.md:10]: I couldn’t find any notes on deployment.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.ts",
+  "scripts": {
+    "test": "cat test/nmap-out.xml | ts-node ./src/index.ts"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@jupiterone/jupiter-managed-integration-sdk": "^33.6.0",
```
Comment (aiwilliams) [inline package.json:12]: This seems like a rather heavy dependency for the one function of checking a public IP address 🤔 Perhaps there is a smaller NPM package that does that kind of thing?

File: README.md (hunk 1 above)
Comment (erkangz) [reply README.md]: good catch, fixed

File: package.json (hunk 3 above)
Comment (erkangz) [reply package.json:12]: i'm hoping to use this in a future iteration to create the Host entities (and potentially other entities)

**Pull request merged**